tox.ini

```diff
@@ -306,6 +297,18 @@ envlist =
     {py3.9,py3.12,py3.13}-strawberry-v0.259.0
 
 
+    # ~~~ Network ~~~
+    {py3.7,py3.8}-grpc-v1.32.0
```

Comment:
In the `.github/workflows`, we don't have either of these Python versions. Is that because the line changed in this PR is for the `latest` tests?